src/components/ui/Transactions/Transactions.tsx

```diff
@@ -33,38 +25,40 @@ const Transactions = ({
   transactions,
   onRefundClicked,
 }: {
-  transactions: Transaction[];
+  transactions: LocalPaymentAttempt[];
   onRefundClicked: (paymentAttemptID: string) => void;
 }) => {
   return (
     <table>
       <tbody>
         {transactions.map((transaction, index) => (
           <tr key={index} className="border-b group">
-            <td className="text-[0.813rem] pb-2 pt-2 leading-6">{format(transaction.occurredAt, 'MMM do, yyyy')}</td>
-            <td className="pl-6 pb-2 pt-2 text-right">
+            <td className={classNames('text-[0.813rem] pb-2 leading-6', { 'pt-2': index !== 0 })}>
+              {format(transaction.occurredAt, 'MMM do, yyyy')}
+            </td>
+            <td className={classNames('pl-6 pb-2 text-right', { 'pt-2': index !== 0 })}>
               <span className="mr-2 text-sm tabular-nums font-medium leading-6">
                 {new Intl.NumberFormat(navigator.language).format(Number(transaction.amount.toFixed(2)))}
               </span>
             </td>
-            <td>
+            <td className={classNames('pb-2', { 'pt-2': index !== 0 })}>
               <span className="text-greyText font-normal text-[0.813rem] leading-6">{transaction.currency}</span>
             </td>
-            <td className="pl-6 pb-2 pt-2 ">
+            <td className={classNames('pl-6 pb-2', { 'pt-2': index !== 0 })}>
               <div className="flex flex-row items-center">
                 <span className="mr-2">
                   <PaymentMethodIcon paymentMethod={transaction.paymentMethod}></PaymentMethodIcon>
                 </span>
                 <span className="text-sm leading-6">{transaction.processor}</span>
-                {transaction.paymentMethod === PaymentMethod.Card && transaction.last4DigitsOfCardNumber && (
+                {transaction.paymentMethod === LocalPaymentMethodTypes.Card && transaction.last4DigitsOfCardNumber && (
                   <div className="text-sm ml-1 items-center flex">
                     <span className="text-[0.375rem] mr-1">&#8226;&#8226;&#8226;&#8226;</span>
                     <span>{transaction.last4DigitsOfCardNumber}</span>
                   </div>
                 )}
               </div>
             </td>
-            <td className="pl-6 pb-2 pt-2 leading-6">
+            <td className={classNames('pl-6 pb-2 leading-6', { 'pt-2': index !== 0 })}>
```

Comment:
You might want to try if you'd achieve the same behaviour of the padding but without using `classNames` helper with just using the `gap-y-2` in the parent (`<tr>` element). It would look a bit cleaner and remove a bit of boilerplate.

It's fine to leave it if gap doesn't work (docs [here](https://tailwindcss.com/docs/gap))

Reply:
I tried this and it didn't work unfortunately 😞

Reply:
Okay, no worries